I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/locations.rpy", line 150, in script call
    call world_map
  File "game/locations.rpy", line 651, in script call
    call event_calls
  File "game/emma_scenes.rpy", line 607, in script call
    call before_action(EmmaX, "fondle_breasts", None)
  File "game/actions.rpy", line 211, in script call
    call Top_Off(Girl)
  File "game/stripping.rpy", line 1256, in script call
    call shift_focus (Girl)
  File "game/utilities.rpy", line 3306, in script
    if Girl.location == bg_current:
  File "game/utilities.rpy", line 3306, in <module>
    if Girl.location == bg_current:
AttributeError: 'NoneType' object has no attribute 'location'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/locations.rpy", line 150, in script call
    call world_map
  File "game/locations.rpy", line 651, in script call
    call event_calls
  File "game/emma_scenes.rpy", line 607, in script call
    call before_action(EmmaX, "fondle_breasts", None)
  File "game/actions.rpy", line 211, in script call
    call Top_Off(Girl)
  File "game/stripping.rpy", line 1256, in script call
    call shift_focus (Girl)
  File "game/utilities.rpy", line 3306, in script
    if Girl.location == bg_current:
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/ast.py", line 1898, in execute
    if renpy.python.py_eval(condition):
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/utilities.rpy", line 3306, in <module>
    if Girl.location == bg_current:
AttributeError: 'NoneType' object has no attribute 'location'

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Rogue-Like 0.997f2
Thu Jun  2 14:32:09 2022
